code/game/objects/structures/crates_lockers/closets/secure/medical.dm

```diff
@@ -135,6 +135,9 @@
 	new /obj/item/clothing/mask/gas/explorer(src)
 	new /obj/item/storage/belt/medical(src)
 	new /obj/item/pickaxe(src)
+	new /obj/item/twohanded/binoculars
```

Comment:
```suggestion
	new /obj/item/twohanded/binoculars(src)
```
need to do this for all the new added stuff

Reply:
Oh, thanks, i am still new to this stuff.